src/components/dashboard.js

```diff
@@ -1,7 +1,214 @@
-import React from "react"
+import React from 'react';
+import { DashboardStyled } from './dashboardStyled';
 
 function Dashboard() {
-  return <div>Hi !!</div>
+  return (
+    <DashboardStyled>
+      <div class="dashboard-wrap row">
+        <nav class="col-2 sidebar">
+          <ul class="nav flex-column">
+            <li class="nav-item mb-5 mt-3">
+              <a class="active" href="#">
+                <img
+                  class="img-fluid"
+                  src="img/svgs/ccn-logo-white.svg"
+                  alt="code clan logo"
+                />
+              </a>
+            </li>
+            <li class="nav-item mb-5 active-icon">
+              <a class="nav-link" href="#">
```

Comment:
Codacy found an issue: [The href attribute requires a valid value to be accessible. Provide a valid, navigable address as the href value. If you cannot provide a valid href, but still need the element to resemble a link, use a button and change it with appropriate styles. Learn more: https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/anchor-is-valid.md](https://app.codacy.com/gh/codeclannigeria/codeclannigeria-frontend/pullRequest?prid=5730659)